packages/nestjs-redis/README.md

```diff
@@ -0,0 +1,52 @@
+# NestJS Redis
+
+Redis, but for NestJS
+
+## Installation
+
+```sh
+yarn add @voiceflow/nestjs-redis ioredis
+yarn add -D @types/ioredis
+```
+
+## Usage
```

Comment:
it may be good to document the other exports.
specifically, adding a note that once `RedisModule` is globally registered you can use `RedisService` anywhere without needing to import `RedisModule` again.